csharp-package/brainflow/brainflow/brainflow_input_params.cs

```diff
@@ -64,6 +64,16 @@ public class BrainFlowInputParams
         /// </summary>
         [DataMember]
         public string file;
+        /// <summary>
+        /// metric to caclulate
+        /// </summary>
+        [DataMember]
+        public int metric;
+        /// <summary>
+        /// classifier to use
+        /// </summary>
```

Comment:
why is it here? there is another class for ML model params

Reply:
Input params needs to save metric

Reply:
no, it saved in another class. Moreover BrainFlowInputParams in another dll from ML API

Reply:
you should change function which parses cmd arguments in test instead, you can make it like here https://github.com/brainflow-dev/brainflow/blob/master/tests/cpp/ml_demo/src/eeg_metrics_ci.cpp#L19
or even better to change it to smth like

```
bool parse_args (int argc, char *argv[], struct BrainFlowInputParams *params, struct BrainFlowModelParams *model_params);
```

Reply:
If I'll do that, I'll add an empty constructor for all of them just to make it easier that fine

Reply:
hm... its bad to add empty constructor because these 2 fields are required. You can create an object of BrainFlowModelParams with wrong\predefined values and override them in parse arguments, Or do it like in cpp test 

```
 bool parse_args (int argc, char *argv[], struct BrainFlowInputParams *params, int *board_id,
    int *metric, int *classifier)
```

Reply:
I'll use bad params.